locales/translations.json

```diff
@@ -148,6 +156,8 @@
   "dateDisabled": "Date disabled",
   "hostAckModalTitle": "Recommendation has been disabled for:",
   "systemName": "System name",
+  "noSystemReboot": "System reboot is <strong> not </strong> required",
+  "systemReboot": "System reboot <strong> is </strong> required",
```

Comment:
looks like this could be the case where yah have this string once, and pass in the `is` / `is not` 🤔 BUT ALSO period at the end of this